presto-main/src/main/java/com/facebook/presto/sql/planner/planPrinter/IOPlanPrinter.java

```diff
@@ -546,7 +546,12 @@ private String getVarcharValue(Type type, Object value)
             if (type instanceof BooleanType) {
                 return ((Boolean) value).toString();
             }
-            throw new PrestoException(NOT_SUPPORTED, format("Unsupported data type in EXPLAIN (TYPE IO): %s", type.getDisplayName()));
+
```

Comment:
We need to keep the throw exception as we are not changing the existing behavior